### DISCUSSION TITLE
Subnet 4via6 Support

### DISCUSSION THREAD

**Discussion by dcd-projectrose** [hassio-addons/app-tailscale#210] [Ideas]
Can there be support added for 4via6 Subnets?

**Comment by lmagyar** [comment]
Hmmm, this could help with colliding/overlapping subnets, especially in case of non-userspace-networking. #201

Sg. like adding an optional config parameter `ip4viaip6_site_id` and if set then advertise all IPv4 addresses like IPv6 with:
```
tailscale debug via <ip4viaip6_site_id> <IPv4 subnet>
tailscale up --advertise-routes=fd7a:115c:a1e0:b1a:0:<ip4viaip6_site_id>:<IPv4 subnet IP in hex>/<IPv4 subnet mask + 96>
```

Note: I'm just another user.

**Comment by ibbit** [comment]
Are there any plans to implement this setting?
I'm facing a overlapping IP Subnets on 2 customer, now I deactivate and activate the subnets.
It would be nice to use this existing "feature".

**Reply by lmagyar** [reply]
There ~is~ (ooops, it got stale and closed) was a PR #562, I commented a lot on it, now I rebased it onto my fork, I will finish it when I have time, but it takes a long time on the official add-on to get sg. merged.
